src/main/java/org/opentripplanner/routing/graph/GraphIndex.java

```diff
@@ -78,6 +78,9 @@
 
     /** maximum distance to walk after leaving transit in Analyst */
     public static final int MAX_WALK_METERS = 3500;
+    
+    /** Should stop clusters be created by parent stations instead of by proximity?  */
+    public static final boolean CLUSTER_BY_PARENT_STATIONS = false;
```

Comment:
I wanted to make it configurable, yes, and took a shortcut by making this constant. But I see now how I can put it in the router-config, so I will do that instead.